tests/Akkling.Tests/StreamsApi.fs

```diff
@@ -132,4 +131,38 @@ let ``Expand must pass through elements unchanged when there is no rate differen
         subscriber.RequestNext i |> ignore
     
     subscriber.Cancel() |> ignore
-   
\ No newline at end of file
+   
+[<Fact>]
+let ``InjectFirst must emits the first element every given interval`` () = test (config()) <| fun tck ->
+    use mat = tck.Sys.Materializer()
+        
+    [1, Some 1; 2, Some 2; 1, Some 11; 2, Some 22]
+    |> Source.ofList
+    |> Source.pulse true (TimeSpan.FromMilliseconds 250.0)
+    |> Source.groupBy 100 fst
+    //|> SubFlow.injectFirst (TimeSpan.FromMilliseconds 100.0) (fun (i, _) -> i, None)
+    |> SubFlow.mergeSubstreams
+    |> Source.take 4L
+    |> Source.runFold mat (fun acc x -> x :: acc) []
+    |> Async.RunSynchronously
+    |> List.rev
+    |> equals [
+        1, Some 1 
+        1, None
+        1, None
+        2, Some 2
+        1, None
+        2, None
+        1, None
+        2, None
+        1, Some 11
+        1, None
+        2, None
+        1, None
+        2, None
+        2, Some 22
+        1, None
+        2, None
+        1, None
+        2, None
+       ]
```

Comment:
@Horusiath This test hangs forever. The cause is `pulse` - if I remove it, the test finishes (it' not green though). What do you think can it be? A deadlock?